postprocessing/janus-pp-rec.c

```diff
@@ -138,6 +150,15 @@ static int audioskew_th = DEFAULT_AUDIO_SKEW_TH;
 #define DEFAULT_SILENCE_DISTANCE 100
 static int silence_distance = DEFAULT_SILENCE_DISTANCE;
 
+#define DEFAULT_RESTAMP_MULTIPLIER 0
+static int restamp_multiplier = DEFAULT_RESTAMP_MULTIPLIER;
```

Comment:
If I understand correctly this parameter is in "thousandths".
e.g. 50 means that the current latency must be greater than 50 / 1000 = 5% of the current avg in order to trigger the compensation.

Reply:
given that the current latency is greater than the threshold of course

Reply:
Yes. Your example is correct.
Maybe I should rename it to something else :thinking: ... any suggestions?

Reply:
I can not think of a better name.
I guess we could leave as it is, since the equivalent CLI option (`restamp`) has been already described.